src/components/remotecontrol/remotecontrol.js

```diff
@@ -13,9 +13,11 @@ import * as userSettings from '../../scripts/settings/userSettings';
 import cardBuilder from '../cardbuilder/cardBuilder';
 import itemContextMenu from '../itemContextMenu';
 import '../cardbuilder/card.scss';
+import '../../elements/emby-button/emby-button';
 import '../../elements/emby-itemscontainer/emby-itemscontainer';
```

Comment:
`emby-itemscontainer` can be removed (no direct use in `remotecontrol.js`), but I didn't want to risk :sweat_smile: